### ISSUE TITLE
Default button does not work when in Live Preview

### ISSUE THREAD

**Issue by johndwells** [marionnewlevant/craft-defaultcolor#1]
Clicking on 'set to default' button while in Live Preview does not do anything. This is the case when the field is on its own and within a Matrix.

**Comment by marionnewlevant** [comment]
Seems to be a problem with colors in general

**Comment by johndwells** [comment]
I've just tested this again on Craft 1.3.2496, and when in Live Preview mode I can select a new color, it's just the "set to default" that doesn't work. I've also tested with a simple Color fieldtype, and again picking the color works.

I'm running OSX if that makes a difference, the color picker seems to be a native OS tool.

But is it not working for you? Should we report this to P&T as a core bug?

**Comment by marionnewlevant** [comment]
which browser? different browsers have different color pickers. And it does
work for you when not in Live Preview?

I did report to P&T what I was seeing:
Using firefox 27.0.1 and latest Craft:
Craft 1.3.2496
Released on 2/27/2014

I have a Color field. Works fine normally, but I don't get the color
picker in Live Preview.

And they confirmed that as a bug.

I will poke into this further, and see what I can figure out.
